smart-contracts/advanced/fevm-indexers.md

```diff
@@ -0,0 +1,91 @@
+---
+description: >-
+    FEVM Indexers allow users and developers to query Ethereum chain data in an extremely quick manner. Learn what EVM indexers are available on Filecoin and how to use them through existing data providers.
+---
+
+# FEVM Blockchain Indexers
+> *Not to be confused with [IPNI Indexer](https://docs.filecoin.io/storage-providers/architecture/network-indexer)*
+
+Blockchain indexers are used for accessing blockchain data efficiently. They process and organize storage-optimized raw blockchain data into retrieve-optimized and well-queryable formats. This benefits developers and users looking to retrieve specific information because they don't need to:
+
+1. Run their own [archival node](https://docs.filecoin.io/networks/mainnet/rpcs).
+2. Parse entire blockchain histories to crawl for events that might not exist for thousands of [tipsets](https://docs.filecoin.io/basics/the-blockchain/blocks-and-tipsets#tipsets).
+3. Spend significant time required to retrieve data from the blockchain node.
+4. Determine complex interconnections between smart contracts.
+5. Spend substantial compute power to calculate advanced queries.
+
+Additionally, blockchain indexers provide a better developer experience by leveraging well-known API standards and query languages like GraphQL.
+
+## The Graph
+
+[The Graph](https://thegraph.com) is a decentralized protocol for indexing blockchain data. It enables developers to build and publish custom open APIs, known as subgraphs, that applications can query to retrieve blockchain data using GraphQL in a time-efficient manner.
+
+### Glossary
+
+- **Subgraphs**: Customizable schemas that define how to index data from specific blockchain smart contracts and events.
+- **GraphQL**: A query language that allows clients to request exactly the data they need, making data fetching more efficient.
+
+### Querying Subgraphs on Filecoin FEVM
+
+There are many ways to query existing subgraphs, including numerous well-known libraries for [JavaScript](https://thegraph.com/docs/en/querying/querying-from-an-application/) and [Python](https://thegraph.com/docs/en/querying/querying-with-python/). But even without any third-party tooling, querying a subgraph is no more complicated than querying [RPC nodes](https://docs.filecoin.io/reference/json-rpc). The only complexity is that you have to know the schema of the subgraph beforehand, similar to knowing SQL database tables and columns before being able to query them. Luckily, The Graph provides several ways to discover the subgraph schema. The most convenient one is called the ["Playground"](https://graphql.org/blog/2020-04-03-graphiql-graphql-playground/), and it is available upon a GET request to the subgraph query URL. Alternatively, you may use the discovery method that exists on every subgraph, called the [Introspection Query](https://graphql.org/learn/introspection/).
+
+### Developing Subgraphs on Filecoin FEVM
```

Comment:
LGTM. @trruckerfling need some help with merging.

Non-blocking nit: The documentation itself is limited in scope, so I would actually consider simplifying these sections even further (by removing the sentence `Although it is not...`)